cmd/rosetta-dispatcher-server/main.go

```diff
@@ -0,0 +1,297 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/labstack/echo/v4"
+	"github.com/ziflex/lecho/v2"
+	"io/ioutil"
+	"net/http"
+	url2 "net/url"
+	"os"
+	"os/signal"
+	"sort"
+	"time"
+
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/optakt/flow-dps-rosetta/service/identifier"
+	"github.com/rs/zerolog"
+	"github.com/spf13/pflag"
+)
+
+const (
+	success = 0
+	failure = 1
+)
+
+type Spork struct {
+	First      uint64
+	Last       uint64
+	ProxyTaget *middleware.ProxyTarget
+}
+
+type SporkList []Spork
+
+func NewSporkList(addresses []string, firsts []int64, lasts []int64) (SporkList, error) {
+	// TODO: Add tests for validating sporks configuration
+
+	// Check past sporks configuration
+	if len(addresses) != len(firsts) || len(addresses) != len(lasts) {
+		return nil, fmt.Errorf("data length mismatch")
+	}
+
+	if len(addresses) < 2 {
+		return nil, fmt.Errorf("at least two sporks must be provided")
+	}
+
+	sporksTable := make(SporkList, len(addresses))
+
+	for i, sporkAddress := range addresses {
+		sporksTable[i].First = uint64(firsts[i])
+		sporksTable[i].Last = uint64(lasts[i])
+
+		url, err := url2.Parse(sporkAddress)
+		if err != nil {
+			return nil, fmt.Errorf("spork address %d is invalid: %w", i, err)
+		}
+		if sporksTable[i].First >= sporksTable[i].Last {
+			return nil, fmt.Errorf("spork %d last height is not greater than first", i)
```

Comment:
```suggestion
			return nil, fmt.Errorf("spork %d last height (%d) is lesser than first (%d)", i, sporksTable[i].Last, sporksTable[i].First)
```

Also IMO calling them `Start` and `End` would be clearer than `First` and `Last`. Any particular reason for using `First` and `Last` instead?